internal/core/ports/runtime.go

```diff
@@ -38,7 +38,7 @@ type Runtime interface {
 	DeleteScheduler(ctx context.Context, scheduler *entities.Scheduler) error
 	// CreateGameRoomInstance Creates a game room instance on the runtime using
 	// the specification provided.
-	CreateGameRoomInstance(ctx context.Context, schedulerId, gameRoomName string, spec game_room.Spec) (*game_room.Instance, error)
+	CreateGameRoomInstance(ctx context.Context, scheduler entities.Scheduler, gameRoomName string, spec game_room.Spec) (*game_room.Instance, error)
```

Comment:
I think it's better to use the reference/pointer of the scheduler here.

Reply:
sure